tests/event_periodic_callback/main.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Periodic Callback test application
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "event/periodic_callback.h"
+
+static void _event_cb(void *ctx)
+{
+    puts(ctx);
+}
+
+int main(void)
+{
+    event_periodic_callback_t a, b, c;
+
+    event_periodic_callback_init(&a, ZTIMER_MSEC, EVENT_PRIO_MEDIUM, _event_cb, "event A");
+    event_periodic_callback_init(&b, ZTIMER_MSEC, EVENT_PRIO_MEDIUM, _event_cb, "event B");
+    event_periodic_callback_init(&c, ZTIMER_MSEC, EVENT_PRIO_MEDIUM, _event_cb, "event C");
```

Comment:
You use the queues of `event_thread` here, but this is application code, so maybe you saw the link error and thought it was a missing dependency?